go/src/koding/klientctl/ctlcli/ctlcli.go

```diff
@@ -116,9 +116,10 @@ func ExitErrAction(f ExitingErrCommand, log logging.Logger, cmdName string) cli.
 		}
 
 		Close()
-		ExitFunc(exit)
```

Comment:
i am not sure why we added os.exit here. It prevents running AfterFunc's of any command.